TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx

```diff
@@ -22,12 +21,12 @@ import {
     RadioGroup,
     RadioLine,
     Slider,
-    TabBar,
     TextSlider,
     OutlinedTextButton,
 } from '@/Components';
+import { TabBar } from '@/Components/ControlBar';
```

Comment:
🙈 Could this be imported from `@/Components` instead?

Reply:
I moved TabBar within the ControlBar folder as it think it fits that area nicer (and the tabs are in there) so it imports from the ControlBar index file.

Seeing as the ControlBar doesn't actually control anything any more I will rename it

Reply:
Oh wait export ControlBar index in Components index so you're right - can just import from Components